conda_forge_tick/update_upstream_versions.py

```diff
@@ -417,3 +418,26 @@ def main(
         n_jobs=n_jobs,
         package=package,
     )
+
+
+def main_find_latest_feedstock_version(
+    name: str = None,
+    node_attrs_file: str = None,
```

Comment:
This is not a valid type annotation.

```suggestion
    name: Optional[str] = None,
    node_attrs_file: Optional[str] = None,
```